src/Data/List/Relation/Binary/Lex.agda

```diff
@@ -0,0 +1,92 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Lexicographic ordering of lists
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.List.Relation.Binary.Lex where
```

Comment:
I have to confess I'm a bit confused by this reorganisation. To me it would make more sense to lift all properties up into this module, rather than just some of them? This would be more in keeping with other binary relations over lists where we have core modules just containing the definition?

You could then get rid of `Lex.Strict.Core` and move all four of the basic definitions into `Lex.Core`?